### PR TITLE
Implemented Account Selection Window and added registration navigation

### PR DESCRIPTION
**Pull request by thanmaiKmandala**
**CHANGES**

pages/index.vue: 
- Added a "Register Now" button with logic design behind it. Button is linked to the accountSelectionWindow.vue
- Added a hover effect to the button to make it easier for user experience

pages/accountSelectionWindow.vue: 
- New file created for this issue. Basically opens up a window that gives 4 options: You can create either a "Parent", "Student", "Faculty" class or you can click on "Close" to close the window.
- Added some navigation logic to make it flow from the accountSelectionWindow to the registration forms for each respective account type. (ex. Clicking on Parent in the window will take you to the parent registration form)

docker-compose.yml:
- Updated port mapping for reading-mlk (3000:3000)
- Set database service port to 5432:5432 instead of using the environment variable $POSTGRES_PORT
- If you are running with "npm run dev" and using the docker desktop, you don't need to worry about this.

### CODE REVIEW

**Comment by Duc-T-Nguyen** [comment]
Note: 
- the create parent and create faculty buttons should not navigate to createparent and createfaculty. Those pages are not the pages we want to navigate during registration and should be instead updated to navigate to parentform and facultyform respectively.
- also we do not need to have the create student button 
- the createparent, createfaculty, and createstudent pages are for admins accounts to exclsuively have access to (requested by Rae to personally add parents, faculty, and students)

File: docker-compose.yml
Comment (OfekShaltiel-UTD) [inline docker-compose.yml:1]: Please do not change the docker-compose file. The Docker container is supposed to be Standardized for everyone in EPICS to run on 11111:5432 (or well 11111:54xx in the future). We can fix this in lab.

**Review by OfekShaltiel-UTD — CHANGES_REQUESTED** [review]
Added code quality comments that should be addressed before the end of the sprint.

File: docker-compose.yml
[the remark this answers is not in the document]
Comment (OfekShaltiel-UTD) [reply docker-compose.yml:1]: Edit: Misread the file, so I'll talk to Taz about the new app service on the docker-compose, but as regards to your pr message, we'll keep the db port as 11111:5432 for now.

**Review by Duc-T-Nguyen — APPROVED** [review]
pugified template, implemented requirements, and complete other issues present in initial pull request

**Pull request merged**